test/heterogeneous_ice_nucleation_tests.jl

```diff
@@ -0,0 +1,53 @@
+import Test
+
+import Thermodynamics
+import CloudMicrophysics
+import CLIMAParameters
+const CP = CLIMAParameters
+
+const TT = Test
+const TD = Thermodynamics
+const CMT = CloudMicrophysics.CommonTypes
+const CMI = CloudMicrophysics.HetIceNucleation
+
+include(joinpath(pkgdir(CloudMicrophysics), "test", "create_parameters.jl"))
+FT = Float64
```

Comment:
I'd try to avoid defining a global non-constant type variable, since this opens the door for major performance hits

Reply:
This is inside our tests. Maybe it's ok to do `const FT = Float64`?

Reply:
I'll try to do something better. But it will be a separate PR